### ISSUE TITLE
Clothing shop not returning items when selecting NONE

### ISSUE THREAD

**Issue by AngryHerring** [AsYetUntitled/Framework#570]
### Expected behaviour
When selecting "REMOVE VEST" or "REMOVE BACKPACK", and closing the shop in a normal fashion, your vest and backpack should be returned.

### Actual behaviour
The shop doesn't return VEST or BACKPACK

### Steps to reproduce the behaviour
1. Currently have an existing VEST or BACKPACK on you.
2. Open clothing shop.
3. Select "REMOVE _X" (Vest or backpack).
4. Close the shop (Exit).
5. Doesn't return back to previous vest OR backpack

### [RPT]
Server .rtp: [https://pastebin.com/hZnvewbt](url)
Client .rtp: [https://pastebin.com/0MKtcuLb](url)

----

Mission version: 5.0

**Comment by klmunday** [comment]
Hello thanks for the report
I have noticed you are using modified files, if possible could you please test this with fresh files to ensure this is not an error caused by modifications?
thanks

**Comment by AngryHerring** [comment]
@klmunday Yes, I've indeed tested this with clean framework from github aswell. The same bug occurs. Please note that if you attempt it with headgear, clothing or glasses the script will return the items correctly!

**Comment by AngryHerring** [comment]
Here is a video demonstrating this bug (clean framework):
https://youtu.be/1KyKYIcxuBU

**Comment by BlackFisch** [comment]
Honestly, I have no problem with the option "Remove X" to actually **remove** the item. I (for my part) never really interpreted it as such, that the item is given back to you (hence the name "remove").
That doesn't mean I don't support maybe a second option to select "No Change" (or sth like that).

Remove X -> actually removed the item from your inventory
No Change -> gives you the old item back

**Comment by klmunday** [comment]
Yes I would agree, if a person selects `remove backpack` and then proceeds to cancel it *should* give the old backpack back, good spot

**Comment by BlackFisch** [comment]
> 
> 
> Yes I would agree, if a person selects `remove backpack` and then proceeds to cancel it _should_ give the old backpack back, good spot

sorry, am i missing something? do you mean if you select remove and cancel the gear disappears? I'll check that :D

**Comment by klmunday** [comment]
That's what was happening in the video. Or am I misinterpreting this issue?

**Comment by BlackFisch** [comment]
> 
> 
> That's what was happening in the video. Or am I misinterpreting this issue?

ah, my bad. thought he clicked buy.... my bad tho
